tools/md_plotting/paracords_plot.xml

```diff
@@ -0,0 +1,120 @@
+<tool id="par_coord_plotting" name="Parallel Coordinates Plot" version="0.1">
+    <description>of tabular data</description>
+    <requirements>
+        <requirement type="package" version="3.7">python</requirement>
+        <requirement type="package" version="0.22.0">pandas</requirement>
+        <requirement type="package" version="3.1.1">plotly</requirement>
+    </requirements>
+    <version_command>echo $version</version_command>
+    <command detect_errors="aggressive"><![CDATA[
+    python '$plot_script'
+]]>
+    </command>
+    <configfiles>
+        <configfile name="plot_script">
+<![CDATA[
+import sys
```

Comment:
Just saying, no preferences. You could put this python code into its own python file and not mess with cheetah. The reason we chose to do it inline in sklearn was that we had too many parameters to pass around - this is not the case here.

Please ping me if this is ready for review!

Reply:
If you're Okay with this, I would like to keep current style. :)
I'll try the other python style in future tools.